tests/DependencyInjection/SonataNotificationExtensionTest.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NotificationBundle\Tests\DependencyInjection;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\NotificationBundle\DependencyInjection\Compiler\NotificationCompilerPass;
+use Sonata\NotificationBundle\DependencyInjection\SonataNotificationExtension;
+use Sonata\NotificationBundle\SonataNotificationBundle;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class SonataNotificationExtensionTest extends TestCase
+{
+    /**
+     * @var ContainerBuilder
+     */
+    private $container;
+
+    protected function tearDown()
+    {
+        unset($this->container);
+    }
+
+    public function testEmptyConfig()
+    {
+        $container = $this->getContainerBuilder([
+            'MonologBundle' => 'Symfony\Bundle\MonologBundle\MonologBundle',
```

Comment:
maybe use `MonologBundle::class` instead of `Symfony\Bundle\MonologBundle\MonologBundle`

Reply:
Yes please, apply it for all strings that can be replaced by ::class

Reply:
Ok, sure
But those classes are not exist, because they are not included in `require` section of composer.json, and PhpStorm will highlight this as error

Reply:
https://3v4l.org/eXMYc

Maybe phpStorm will complain, but it does work.

Reply:
@jordisala1991 yes, it works. i just tried to avoid that highlightings =) i've pushed changes, waiting for travis